src/modules/billing/controllers/batches.js

```diff
@@ -110,9 +110,21 @@ const getBatchInvoiceDetail = async request => {
 const deleteBatchInvoice = async (request, h) => {
   const { batch } = request.pre;
   const { invoiceId } = request.params;
+
+  let originalInvoiceId = null;
+  let rebillInvoiceId = null;
+
+  if (request.query) {
+    originalInvoiceId = request.query.originalInvoiceId;
+    rebillInvoiceId = request.query.rebillInvoiceId;
+  }
   try {
-    // Delete the invoice
-    await batchService.deleteBatchInvoice(batch, invoiceId);
+    if (originalInvoiceId && rebillInvoiceId) {
+      await batchService.deleteBatchInvoice(batch, invoiceId, originalInvoiceId, rebillInvoiceId); /* delete both related bills */
+    } else {
+      // Delete the invoice
+      await batchService.deleteBatchInvoice(batch, invoiceId);
+    }
```

Comment:
You don't need the if statement here if you are checking the values exists in the batch-service. 

You could just leave this line below and get rid of the rest

```await batchService.deleteBatchInvoice(batch, invoiceId, originalInvoiceId, rebillInvoiceId);```